xcodeproj/internal/xcschemes/xcschemes_execution.bzl

```diff
@@ -317,7 +317,7 @@ def _write_schemes(
             _add_execution_action(
                 action,
                 action_name = action_name,
-                id = target.id,
+                id = getattr(target, "id", None),
```

Comment:
I think checking `is_path` in an inline `if else` would be more performant.

Reply:
I dont have access to `is_path` in here without adding another argument. I moved the `getattr` to be outside the for loop though

Reply:
But `target` is `launch_target`, which has `.is_path`, right?

Reply:
It wont always be, it looks like this is called for test targets too which wouldn't have `is_path`. But maybe I'm missing something

Reply:
Ahh. Can we change it so we pass in `id` then? We already do an `if` on `is_path`, so we can assign to the variable in both paths there.

Reply:
Updated!